### ISSUE TITLE
Non-Zero Y-Axis for charts

### ISSUE THREAD

**Issue by 123dev** [Graylog2/graylog2-server#2337]
### Problem description
Is there a way to generate Graylog charts with non-zero y axis?
We're trying to chart some data, where all values are > 25,000 but the delta is small (+/- few hundred)
and the chart looks almost flat, because Y-Axis is always starts at zero.

Is it possible to set it to non-zero?
Would be great also if the minimum value can automatically be picked up as Y-axis baseline.
If yes how? if no, would you consider adding this as a feature in a future release?

### Steps to reproduce the problem
Chart a field that has values like this.
25000, 25056, 25002, 25102, 25044

### Environment

* Graylog Version: 1.3.2 AWS Image

Thanks

**Comment by edmundoa** [comment]
Hi,

This is currently not possible to do with Graylog, but it makes sense, so I think we can add an option for that.

Out of curiosity, is your widget displaying a search histogram or some field value? In the second case, you might try to add different versions of the widget using different renderers and interpolation, maybe that will help making the small deltas slightly more visible.

**Comment by 123dev** [comment]
Thanks Edmundoa.

Good to know that you would consider it as an option in a future release.

The widget is displaying field value, and playing with interpolation is not making positive difference in assessing the data.

By renderers, do you mean the type? sadly that too did not improve the data interpretation.

Thanks again

**Comment by edmundoa** [comment]
Yes, by renderer I meant changing the type of graph. Sorry to hear that it didn't help.

**Comment by Skeen** [comment]
What is the state of this in late 2018?
